### PR TITLE
Add scrollbars to drawers; shorten dividers

### PR DESCRIPTION
**Pull request by edlu77**

### CODE REVIEW

File: projects/ccf-rui/src/app/app.component.scss [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
     ccf-drawer {
       width: 25.5rem;
+      background: whitesmoke;
+
+      ::ng-deep .cff-drawer-inner-container {
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/app.component.scss:14]: Why is this needed? Why not put scrolling on the sidebars themselves?

File: projects/ccf-rui/src/app/app.component.scss [hunk 2]
```diff
@@ -9,6 +9,12 @@
 
     ccf-drawer {
       width: 25.5rem;
+      background: whitesmoke;
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/app.component.scss:12]: No hardcoded colors here!

File: projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss [hunk 3]
```diff
@@ -1,7 +1,8 @@
 :host {
   display: flex;
   flex-direction: column;
-  height: 100%;
+  padding-left: 1.5rem;
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss:4]: Padding will not work as the `hr` elements need to span from the entire sidebar. That is the purpose of the `& > :not(hr)` selector

File: projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss [hunk 4]
```diff
@@ -1,7 +1,8 @@
 :host {
   display: flex;
   flex-direction: column;
-  height: 100%;
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss:4]: Use a `min-height: 100%` to make sure the sidebar takes up all the height (for background purposes mainly). Then set `overflow-y: auto` to get scrolling

File: projects/ccf-rui/src/styles.scss [hunk 5]
```diff
@@ -50,22 +50,18 @@ mat-icon {
 
 // Reference: https://css-tricks.com/custom-scrollbars-in-webkit/
 ::-webkit-scrollbar {
```
Comment (axdanbol) [inline projects/ccf-rui/src/styles.scss:52]: Not sure if you need to change all of this? Have you confirmed with Leonard and/or Bruce? Personally I think it looks quite ugly with these settings.

Comment (axdanbol) [reply projects/ccf-rui/src/styles.scss:52]: Although there are no examples of sidebars scroll in the spec, page 18 shows a scrollbar in the info modal that looks like the ones produced by the previous settings.

Comment (axdanbol) [reply projects/ccf-rui/src/styles.scss:52]: Update: Changes are according to spec.

File: projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss (hunk 3 above)
Comment (axdanbol) [reply projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss:4]: Update: `hr` are not supposed to span entire sidebar

**Review by bherr2 — APPROVED** [review]

**Review by axdanbol — APPROVED** [review]
